app/templates/components/public/session-filter.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="ui segment">
+  {{#each-in tracks as |track|}}
```

Comment:
I think tabs would look better. And when a user presses on a track, don't scroll to it. Instead filter the whole list to display only that track.